src/test/java/io/aiven/kafka/connect/transforms/ExtractTopicTest.java

```diff
@@ -159,7 +223,21 @@ void fieldName_NullOrEmptyValue_NoSkip(final String value) {
     @ParameterizedTest
     @ValueSource(strings = "missing")
     @NullAndEmptySource
-    void fieldName_NullOrEmptyValueOrMissingField_Skip(final String value) {
+    void fieldName_NullOrEmptyValue_NoSkip_Schemaless(final String value) {
+        final Map<String, Object> valueMap = new HashMap<>();
```

Comment:
This and the test `fieldName_NullOrEmptyValueOrMissingField_Skip_Schemaless` could have extra fields in the map.